component/Page/Profile/index.tsx

```diff
@@ -206,20 +206,20 @@ export default function ProfilePage(props: PageProps) {
         <div id="fixed-element-container" className="w-full md:max-w-M lg:max-w-M xl:max-w-M absolute left-1/2 -translate-x-1/2 top-0 pb-4 z-50" style={{
             transition: 'all 0.25s ease'
         }}>
-            <div id='input-message-container' ref={inputMessageContainerRef} className={data?.allow_anon_dm ? `w-full bg-white md:max-w-M lg:max-w-M xl:max-w-M p-2 px-4 fixed flex flex-row gap-[6px] z-[9999]` : `max-w-[375px] p-2 mb-4 ${isLoading ? 'bg-transparent' : 'bg-gray05'} fixed bottom-0 left-12 right-12 flex flex-row gap-[6px] rounded-lg z-[9999]`}>
+            <div id='input-message-container' ref={inputMessageContainerRef} className={data?.allow_anon_dm ? `w-full bg-almostBlack md:max-w-M lg:max-w-M xl:max-w-M p-2 px-4 fixed flex flex-row gap-[6px] z-[9999]` : `max-w-[375px] p-2 mb-4 ${isLoading ? 'bg-transparent' : 'bg-gray05'} fixed bottom-0 left-12 right-12 flex flex-row gap-[6px] rounded-lg z-[9999]`}>
                 {!isLoading && <>
                     {data?.allow_anon_dm ? (
                         <>
                             <Image className="rounded-full self-start" src='/image/anonIcon.svg' alt="anon icon" width={24} height={24} />
-                            <div className="flex flex-grow items-ce bg-gray05 rounded-xl py-1 px-2 items-center">
+                            <div className="flex flex-grow items-ce bg-grey210 rounded-xl py-1 px-2 items-center">
                                 <textarea ref={textAreaRef} autoFocus rows={4} onChange={(e) => {
                                     setMessage(e.target.value)
                                 }} className="bg-transparent min-h-[48px] w-full resize align-top"
                                     style={{ resize: 'none' }}
                                     placeholder="Send me a message 😀" />
                                 <Toggle
                                     checked={true}
-                                    className="bg-white self-start mt-1"
+                                    className="bg-almostBlack self-start mt-1"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous hunk, the changes here are mostly about styling. The background color of the message container and the toggle button has been changed. Again, please ensure that these changes meet accessibility standards, especially in terms of color contrast. 

Overall, the code looks good from a functional perspective. The onClick handlers and other logic remain unchanged. However, it's important to consider accessibility when making style changes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->